CalendarApp/BookingRoom/Controllers/PostController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using BookingRoom.Models;
+using BookingRoom.Models.GoogleCalendar;
+using Google.Apis.Calendar.v3.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using BookingRoom.Models.GoogleEvent;
+using log4net;
+using Google.Apis.Calendar.v3;
+using CalendarGoogleService;
+
+namespace BookingRoom.Controllers
+{
+
+    public class PostController : EventController
+    {
+        public PostController(GoogleService connection)
```

Comment:
Is it possible to rename this to smth closer to the context of usage and its class